hubspot/plugin.py

```diff
@@ -617,8 +630,6 @@ def hubspot_update_contacts(
     Contact IDs that have been updated.
 
 The property names MUST be one of the following:
-
-Default Properties of Type String:
```

Comment:
let's keep these, i suspect code gen will find them helpful.

e.g., when to call .as_datetime etc